judge/views/problem.py

```diff
@@ -294,6 +294,13 @@ def get_paginator(self, queryset, per_page, orphans=0,
                 queryset = queryset.order_by(self.order.replace('name', 'i18n_name'), 'id')
             elif sort_key == 'group':
                 queryset = queryset.order_by(self.order + '__name', 'id')
+            elif sort_key == 'editorial':
+                queryset.annotate(has_public_editorial=Case(
```

Comment:
Is this annotation necessary? We already annotate unconditionally in `get_normal_queryset`.

Reply:
Fixed, good catch, thanks.